pages/linux/st.md

```diff
@@ -0,0 +1,24 @@
+# st
+
+> A simple terminal emulator for the X Window System.
+> More information: <https://st.suckless.org>.
+
+- Open a terminal:
+
+`st`
+
+- Open a terminal with a specific title:
+
+`st -T {{title}}`
+
+- Open a terminal, execute a given command, and write the output to a file:
+
+`st -e {{command}} -o {{path/to/file}}`
```

Comment:
From the manpage:

>  -e command [ arguments ... ]
              st executes command instead of the shell.  If this is used it **must
              be the last option on the command line**, as in xterm / rxvt.  This
              option is only intended for compability, and all the remaining
              arguments are used as a command even without it.

```suggestion
`st -o {{path/to/file}} -e {{command}}`
```

Reply:
Oops, good catch -- thanks! :-)